### ISSUE TITLE
[EN][NPC][Stormwind]Harlan Bagley

### ISSUE THREAD

**Issue by TheRealestMcCoy** [dalaranwow/dalaran-wow#4591]
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
He is running in place?
**Expected behaviour**: Tell us what should happen instead
He should be standing in the shop.
**Steps to reproduce the problem**:

1. Im assuming he was aggroed by a horde player and did or does not reset properly
2. 
3. 

**Include proofs for this behaviour**
![wowscrnshot_031717_010914](https://cloud.githubusercontent.com/assets/12252441/24029671/6e3b2922-0aae-11e7-9bf7-bfe6317893dc.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
[Harlan Bagley](http://www.wowhead.com/npc=1427/harlan-bagley)
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Teridan-dal-wow** [comment]
Hello, @TheRealestMcCoy  is this the first time you see him like that? 
On my server he is inside and not moving. This issue might be solved by simple server restart if that doesn't work respawning the creature should:
``` sql
DELETE FROM `creature` WHERE `guid` = 79716;
INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `spawndist`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `VerifiedBuild`) VALUES
(79716, 1427, 0, 0, 0, 1, 1, 0, 0, -8782.9, 640.188, 97.4065, 5.68977, 180, 0, 0, 1, 0, 0, 0, 0, 0, 0);

DELETE FROM `creature_addon` WHERE `guid` = 79716;
INSERT INTO `creature_addon` (`guid`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `auras`) VALUES
(79716, 0, 0, 0, 1, 0, '');
```
![image](https://cloud.githubusercontent.com/assets/6797611/24039861/6235c14e-0b0f-11e7-8289-94ff0315ecf9.png)

**Comment by TheRealestMcCoy** [comment]
@Teridan-dal-wow I can really say for sure to be honest. This is the first time i noticed it.

**Comment by Rushor** [comment]
hm it's working for me too on ptr - will respawn him as you said (ps: you don't need to add `creature_addon` for every spawn if it contains no useful information)

**Comment by Rushor** [comment]
DB/Creature: Harlan Bagley - Respawn creature in stormwind 

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4591
* commit: ef837634f7e71121b64467dbe6054cc0040d9135

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Rushor** [comment]
the correct fix for it:

Game/Unit: Add attackstart for creatures once the charm fades away + add basic SAI to all creatures with CREATURE_FLAG_EXTRA_CIVILIAN which are spawned in the world

* updates: https://github.com/dalaranwow/dalaran-wow/issues/4613
* commit: 88398faa1a75beba3b21e477de03f9ae7da14168

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TeridanDW** [comment]
Works as intended on PTR.
